roles/ipabackup/library/ipabackup_get_backup_dir.py

```diff
@@ -56,7 +56,12 @@
 '''
 
 from ansible.module_utils.basic import AnsibleModule
-from ipaplatform.paths import paths
+MODULE_IMPORT_ERROR = None
+try:
+    from ipaplatform.paths import paths
+except ImportError as _err:
+    MODULE_IMPORT_ERROR = str(_err)
+    paths = None
```

Comment:
I prefer to set the variable after the fact, like:
```python
try:
    from ipaplatform.paths import paths
except ImportError as _err:
    MODULE_IMPORT_ERROR = str(_err)
    paths = None
else:
    MODULE_IMPORT_ERROR = None
```

Reply:
Done